README.md

```diff
@@ -1,14 +1,18 @@
 # Bro & VAST
 
-This package enables Bro to communicate with [VAST][vast], a scalable plaform
-for network forensics.
+This package enables Bro to leverage the [`bro-to-vast`][bro-vast] tool for
+querying [VAST][vast] from Bro scripts.
+
+VAST is a platform for network forensics at scale. Integrating it into Bro
+allows threat hunters to automate routine tasks, for example using historic
+data to give context to new intelligence feed entries.
```

Comment:
The paragraph about automation sounds goo. But I wouldn't explain VAST here. That's why we have a link. The audience for this repo is the intersection of people who know about Bro and VAST. Maybe instead:

> Combining VAST and Bro allows threat hunters to automate routine tasks, such as correlating new intelligence feed data with context from the past.